impeller/display_list/display_list_dispatcher.cc

```diff
@@ -426,7 +430,26 @@ void DisplayListDispatcher::setColorSource(
       return;
     }
     case flutter::DlColorSourceType::kConicalGradient:
-    case flutter::DlColorSourceType::kRuntimeEffect:
+    case flutter::DlColorSourceType::kRuntimeEffect: {
+      const flutter::DlRuntimeEffectColorSource* runtime_effect_color_source =
+          source->asRuntimeEffect();
+      auto runtime_stage =
+          runtime_effect_color_source->runtime_effect()->runtime_stage();
+      auto uniform_data_sk = runtime_effect_color_source->uniform_data();
+
+      paint_.color_source = [runtime_stage, uniform_data_sk]() {
+        // TODO(bdero): Get rid of the allocation + copy for uniform data.
```

Comment:
Please file an issue for this TODO.

Reply:
Done: https://github.com/flutter/flutter/issues/113714